docs/CNDP_README.md

```diff
@@ -0,0 +1,371 @@
+<!--
+SPDX-License-Identifier: Apache-2.0
+Copyright 2020 Intel Corporation
+-->
+
```

Comment:
Can you please a table of contents (like what I did in the INSTALL.md file (https://github.com/omec-project/upf/blob/master/docs/INSTALL.md?plain=1#L8). It will facilitate the navigation throughout the document, especially for new users.